pkg/agent/controller/networkpolicy/cache.go

```diff
@@ -60,6 +60,10 @@ type rule struct {
 	// The parent Policy ID. Used to identify rules belong to a specified
 	// policy for deletion.
 	PolicyUID types.UID
+	// The metadata of parent Policy. Used to associate the rule with Policy
+	// for troubleshooting purpose (logging and CLI).
+	PolicyName      string
```

Comment:
Question - do we cache NetworkPolicy itself? If so, here we can point to NetworkPolicy?

Reply:
Not yet, that's why it stores the name directly.

Reply:
Ok.